src/sentry/models/dsymfile.py

```diff
@@ -29,6 +35,14 @@
 from sentry.reprocessing import resolve_processing_issue
 
 
+ONE_DAY = 60 * 60 * 24
+ONE_DAY_AND_A_HALF = int(ONE_DAY * 1.5)
+DSYM_MIMETYPES = dict((v, k) for k, v in KNOWN_DSYM_TYPES.items())
+
+
+_proguard_file_re = re.compile(r'(?:^|/)proguard/(?:mapping-)?(.*?)\.txt$')
```

Comment:
@bretthoerner it's supposed to mean that the file can either be `proguard/UUID.txt` or `/proguard/UUID.txt`. Some zip archives use absolute paths. That said, pretty sure that regex is now used on absolute filesystem paths instead of in-archive names so it probably should not matter.